testing/homebank/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor:
+# Maintainer: 7heo <7heo@mail.com>
+pkgname=homebank
+pkgver=5.1.5
+pkgrel=0
+sourcerev=56 # bump pkgrel when you change this. Get it form http://bazaar.launchpad.net/~mdoyen/homebank/5.1.x/changes
```

Comment:
Non-standard global variables (i.e. not defined by abuild) should be prefixed with underscore (i.e. `_sourcerev`).